### ISSUE TITLE
[feature request] Loading bar for citations

### ISSUE THREAD

**Issue by caileyfitz** [CenterForOpenScience/osf.io#2025]
# Steps
1. Enable Zotero for a project
2. Configure Zotero for the project and be sure to link to a folder that has many citations (200+)
3. Go to the overview page to view Zotero citations

# Expected
I expect that the citations would load after a few seconds, and if not, that there would be some sort of indication that they are loading.

# Actual
The widget is blank until 200 citations load. For my folder with 688 citations (and probably more since Zotero also displays attachments as separate citations), it took ~ 20 seconds to load anything in the widget. Initially, I thought that it just wasn't working. It was only when I pulled up the dev console that I saw the GET request was still pending. 

# Suggestions
I would suggest adding some sort of indication or message that citations are loading, because otherwise, users may not stay on the page for long enough to see that the citations do eventually load.

Apparently @harrismendell implemented a loading bar but it is not showing up. May have to do with the fact that @lyndsysimon may not have merged the changes.

**Comment by lbanner** [comment]
@samchrisinger worked to improve that prior to deployment, and I think there are some follow-on considerations on how to improve it.

**Comment by lbanner** [comment]
Any indication @caileyfitz @samchrisinger that this remains an issue?

**Comment by caileyfitz** [comment]
@lbanner Yes, I am still having this problem. Just tried again and it took 19.79 seconds for anything to show up in the widget.

**Comment by lbanner** [comment]
OK
